Dockerize existing sample project

Octoate opened the pull request:
I created a Dockerfile that is containing a multi stage build in a Windows container. The documentation for it is contained in the Dockerfile, however, to build it, just use

> docker build -t inpainter/inpainter Dockerfile

and run it with

> docker run -p 8069:8069 inpainter/inpainter

afterwards. The container should be started and you can access the webserver via "http://localhost:8069".

Octoate commented:
If you need a file that doesn't use the official .NET Framework in a Windows container, please tell it to me and I will create anotherone for Linux based on Mono.

ferib commented:
Hey, Since i am using Linux/Ubunut 16.01 as my server, i would love to see one for Linux based on Mono, thanks

Octoate commented:
Ok... I renamed the original Windows container based Dockerfile to Dockerfile.Windows and added a new one that uses Mono for compilation and running. Runs fine on my system (Docker for Windows), but should also run under Linux and Mac.
